### DISCUSSION TITLE
CentOS7 reaching EOL. Plans for other Linux Distros?

### DISCUSSION THREAD

**Discussion by jaredmcphillen** [AcademySoftwareFoundation/OpenRV#435] [Q&A] [answered]
CentOS 7 is reaching EOL, where it will no longer be supported with updates. And the future of CentOS is ...

Anyway, are there plans to support a different Linux distribution? Which one(s)?

**Comment by aclark4life** [comment]
I remember in 2021 with almost no effort I dropped in Rocky Linux 8 to replace CentOS 8, which reached EOL back then. But with CentOS 7, it looks like you'll need to [get to RL8 first then RL9](https://forums.rockylinux.org/t/migrate-centos-7-to-rocky-linux9/11038/4). Alma is the other path from CentOS to something actively maintained, [similar process described here](https://wiki.almalinux.org/elevate/ELevating-CentOS7-to-AlmaLinux-9.html).

**Comment by jaredmcphillen** [comment]
Thanks. Does OpenRV have any plans to officially support installation/running program on any other Linux distro?

**Reply by bernard-laberge-adsk** [reply, marked as answer]
The plan is to officially support Rocky Linux 8 and Rocky Linux 9.
That being said, we would welcome any community member contribution to support Open RV on other Linux distributions.

**Reply by jaredmcphillen** [reply]
Thanks!
